jane/doc/parallelism-tutorial.md

```diff
@@ -738,70 +847,44 @@ let always_portable : int_or_string @ nonportable -> int_or_string @ portable =
 (We have to write this slightly awkwardly to specify the returned mode of a
 function.)
 
-In fact, we've already seen mode crossing in action. Let's look at the full type
-of `Parallel.fork_join2` as `parallel_kernel.mli` writes it:
-
-```ocaml
-val fork_join2
-  :  t @ local
-  -> (t @ local -> 'a @ contended portable) @ portable
-  -> (t @ local -> 'b @ contended portable) @ portable
-  -> 'a * 'b @ contended portable
-```
+In fact, we've already seen mode crossing in action. We mentioned before when
+talking about [rule 4 of `contended`](#rule-contended-deep) that when
+`Thing.price` projects the `price` field out of a `contended` `Thing.t`, the
+only reason it gets to return an `uncontended` `float` is that `float` crosses
+contention (which is to say, it has kind `immutable_data`).
```

Comment:
Groan. Very sadly, `float : immutable_data` no longer holds. The problem is that `immutable_data` expands to `value mod ... non_float`. So, yeah, not `float`.

Reply:
Thought about hand-waving this a bit but I think it's easiest just to delete the parenthetical. There's already the blurb after the table that introduces the idea that some special types don't have these exact kinds but do cross those modes.